cmd/secrets-provider/main.go

```diff
@@ -51,63 +51,52 @@ func main() {
 		printErrorAndExit(messages.CSPFK009E)
 	}
 
-	// Configure exponential backoff
-	expBackoff := backoff.NewExponentialBackOff()
-	expBackoff.InitialInterval = 2 * time.Second
-	expBackoff.RandomizationFactor = 0.5
-	expBackoff.Multiplier = 2
-	expBackoff.MaxInterval = 15 * time.Second
-	expBackoff.MaxElapsedTime = 2 * time.Minute
-
+	// Configure constant backoff for application container
```

Comment:
what do you mean by `for application container`? this will not happen also from init container?

Reply:
small left over. removing

Reply:
why are they multiplied by 15 and by 2 ?

Reply:
the red means it was removed @oburstein-hub